billy-core-jpa/src/main/java/com/premiumminds/billy/persistence/entities/jpa/JPABusinessEntity.java

```diff
@@ -164,6 +172,11 @@ public void setFinancialID(String id) {
         this.taxId = id;
     }
 
+    @Override
+    public void setFinancialIDCountry(final String isoCountryCode) {
+        this.taxIdCountry = isoCountryCode;
```

Comment:
Taking into account the input argument `isoCountryCode` shouldn't the methods, entity field naming and database column a include the reference to iso country code? Or the oposite, and remove the argument naming and update upstream.
As it is it feels like there's contextual knowledge being lost on this level that either is important and should be kept or should not exist to begin with anywhere.

Reply:
Shoudn't i keep the same naming schema for the new column as the one adopted for the `taxId` column? That's why i keeped `taxIdCountry`.

However i could use `taxIdISOCountryCode`, while the getter and setter would be `get/setFinancialIdISOCountryCode` to keep them consistent with the existing `get/setFinancialID`. Does that seem adequate??

Reply:
Sounds good :+1: